src/pages/learn/index.tsx

```diff
@@ -0,0 +1,943 @@
+              <DocLink to="/bridges/">Bridges</DocLink>
+              <DocLink to="https://weekinethereumnews.com/" isExternal>
+                Week in Ethereum News
+              </DocLink>
+            </DocsContainer>
+          </Section>
+
+          <Section>
+            <h2 id={tocItems[5].id}>{tocItems[5].title}</h2>
+            <p>
+              A big portion of Ethereum’s success is thanks to its dedicated
+              community. There are thousands of inspiring and driven people who
+              help push Ethereum’s vision forward. Come and join us!
+            </p>
+            <CardGrid>
+              <Card
+                title="Community hub"
+                description="Our community includes people from all backgrounds."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.enterprise)}
+                      alt="Illustration of a group of builders working together."
+                    />
+                  </CardImage>
+                  <ButtonLink to="/community/">Explore more</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="How can I get involved?"
+                description="There are technical and non-technical ways to be part of the community."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.dogeComputer)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/community/get-involved/">
+                    How can I get Involved?
+                  </ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Online communities"
+                description="Online communities provide a great opportunity to ask more specific questions or get involved."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.impact)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/community/online/">
+                    Explore communities
+                  </ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+          </Section>
+
+          <Section>
+            <h2 id={tocItems[6].id}>{tocItems[6].title}</h2>
+            <div>
+              <p>Books on Ethereum and Cryptocurrencies:</p>
+              <ul>
+                <li>
+                  <Link to="https://www.goodreads.com/book/show/57356067-the-cryptopians">
+                    The Cryptopians: Idealism, Greed, Lies, and the Making of
+                    the First Big Cryptocurrency Craze
+                  </Link>{" "}
+                  <em>February 22, 2022 - Laura Shin</em>
```

Comment:
We should use `<i>` for italics unless we trying to emphasize the text contained within the tags

```suggestion
                  <i>February 22, 2022 - Laura Shin</i>
```